.gitignore

```diff
@@ -1,3 +1,6 @@
+# pycharm
+.idea/
```

Comment:
Irrelevant to this change

Reply:
Agree, you recommend creating separate branch and PR for those kind of updates?

Reply:
Yes, as a side note: no PR in history was "too short". :)